packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -106,7 +122,11 @@ export async function buildSwingset(
       return;
     }
     if (!config) throw Fail`config not yet set`;
-    const { coreProposals, exportStorageSubtrees } = config;
+    const {
+      coreProposals,
+      clearStorageSubtrees,
+      exportStorageSubtrees = [],
```

Comment:
so `...swingSetConfig` here, and we change which one we pass into `initializeSwingSet` at line 218